runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -1259,13 +1855,13 @@ TR::Register *J9::Power::TreeEvaluator::irdbariEvaluator(TR::Node *node, TR::Cod
    deps->addPostCondition(r11Reg, TR::RealRegister::gr11);
    deps->addPostCondition(metaReg, TR::RealRegister::NoReg);
    deps->addPostCondition(condReg, TR::RealRegister::NoReg);
-
+   
    if (node->getSymbolReference()->getSymbol()->isInternalPointer())
       {
       objReg->setPinningArrayPointer(node->getSymbolReference()->getSymbol()->castToInternalPointerAutoSymbol()->getPinningArrayPointer());
       objReg->setContainsInternalPointer();
       }
-
+   
```

Comment:
Can we remove these extra spaces? It doesn't look like a necessary change.

Reply:
Fixed here:
https://github.com/eclipse/openj9/pull/6234/files#diff-efcdf3eda93e6ae70d4c46dd0bf9b5acL1268

@dchopra001 going to resolve this one on your behalf.